include/alpaka/mem/buf/uniformCudaHip/Copy.hpp

```diff
@@ -364,13 +250,14 @@ namespace alpaka
             Idx m_extentWidth;
 #    endif
             Idx m_extentWidthBytes;
-            Idx m_dstWidth; // required for 3D peer copy
-            Idx m_srcWidth; // required for 3D peer copy
+            Idx m_dstWidth;
+            Idx m_srcWidth;
 
             Idx m_extentHeight;
-            Idx m_dstHeight; // required for 3D peer copy
-            Idx m_srcHeight; // required for 3D peer copy
-
+#    if ALPAKA_DEBUG >= ALPAKA_DEBUG_FULL
+            Idx m_dstHeight;
```

Comment:
Is there a reason we're not putting the width here, too? Also below for the 3D case.

Reply:
width is used somewhere in a call. All attributes which are guarded by the define are only qctive for debugging but are not required for qny api call.